src/plugins/legacy/polygonRoi/toolboxes/polygonRoiToolBox.cpp

```diff
@@ -191,10 +194,6 @@ void polygonRoiToolBox::updateView()
             return;
         }
         medAbstractImageView *v = qobject_cast<medAbstractImageView*>(view);
-        if (!v)
-        {
-            return;
-        }
```

Comment:
Removing these lines can only add a potential bug (if `updateView` is called on a view that is not a `medAbstractImageView` type), so it would be safer to keep them no ?